### PR TITLE
User log in

### PR DESCRIPTION
**Pull request by johannaanden**
### PT Link
[PT Link](https://www.pivotaltracker.com/story/show/162193285)

### User Story
```
As a user
In order to access my account
I would like to be able log in
```
### Actions
Added feature file with feature and scenarios - happy and sad path

### LoFi
Visit PT to see the LoFi
[PT Link](https://www.pivotaltracker.com/story/show/162193285)

### Gems added
No gems added

### Comment 
Task: Switch placeholder link with real link
Pending: Will be fixed with separate PR

### CODE REVIEW

File: features/user_log_in.feature [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    in order to access my account
+    I would like to be able to log in
+
+    Background: 
+        Given the following user exists
+        | email          | password  | password_confirmation |
+        | Don@trump.com  | maga2020  | maga2020              |
+        And I visit login page
+
+    Scenario: User can log in [Happy-path]
```
Comment (oliverochman) [inline features/user_log_in.feature:13]: I want to see a sad path scenario aswell

File: features/step_definitions/assertion_steps.rb [hunk 2]
```diff
@@ -4,6 +4,7 @@
   
 Then("I see {string}") do |content|
     expect(page).to have_content content
+
```
Comment (oliverochman) [inline features/step_definitions/assertion_steps.rb:7]: remove unnecessary line break

**Review by oliverochman — CHANGES_REQUESTED** [review]

File: features/user_log_in.feature (hunk 1 above)
Comment (johannaanden) [reply features/user_log_in.feature:13]: We have a sad path scenarion
```
Scenario: User fills in the login form incorrectly [Sad-path]
        When I click 'Log in'
        Then I see 'Invalid Email or password.'
```

File: features/step_definitions/assertion_steps.rb (hunk 2 above)
Comment (johannaanden) [reply features/step_definitions/assertion_steps.rb:7]: done

**Review by oliverochman — APPROVED** [review]

**Pull request merged**